TODO

```diff
@@ -0,0 +1,13 @@
+// snek TODO
+
+add optional game modifications
```

Comment:
What is specifically meant by that?

Reply:
second apple, power-ups and stuff like that xD